packages/react-components/react-combobox/stories/Dropdown/DropdownClearable.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { Dropdown, Label, makeStyles, Option, shorthands, useId } from '@fluentui/react-components';
+
+const useStyles = makeStyles({
+  root: {
+    display: 'grid',
+    gridTemplateRows: 'repeat(1fr)',
+    justifyItems: 'start',
+    ...shorthands.gap('2px'),
+    maxWidth: '400px',
+  },
+});
+
+export const Clearable = () => {
+  const dropdownId = useId('');
+  const styles = useStyles();
+
+  return (
+    <div className={styles.root}>
+      <Label htmlFor={dropdownId}>Pick a color</Label>
+      <Dropdown clearable aria-labelledby={dropdownId} placeholder="Select a color">
```

Comment:
probably by copy-paste appears this small issue...
not sure if you wanted to use:
-  `aria-labelledby` approach, then we are missing `id` on `<Label>`
- or `htmlFor` then we are missing `id` on dropdown

Reply:
Addressed, thx